contracts/core/src/token/OmniBridgeL1.sol

```diff
@@ -14,7 +16,7 @@ import { OmniBridgeNative } from "./OmniBridgeNative.sol";
  * @notice The Ethereum side of Omni's native token bridge. Partner to OmniBridgeNative, which is
  *         deployed to Omni's EVM.
  */
-contract OmniBridgeL1 is OwnableUpgradeable {
+contract OmniBridgeL1 is OwnableUpgradeable, PausableUpgradeable, OmniBridgeCommon {
```

Comment:
`OmniBridgeCommon` is both `OwnableUpgradeable` and `PausableUpgradeable`, so we shouldn't need to include those inheritances here as well

Reply:
We don't need to but I like showing it explicitly here

Reply:
eh guess it looks nice with just one - ill think about your top level comment too